src/Consolonia.Core/Drawing/DrawingContextImpl.cs

```diff
@@ -797,6 +819,37 @@ private static byte GetBoxPatternFromLineStyle(byte pattern, LineStyle lineStyle
             }
         }
 
+        private void DrawGlyphInfoInternal(Color foregroundColor, GlyphInfo glyphInfo, ConsoleTypeface glyphTypeface,
+            PixelPoint position)
+        {
+            // NOTE: we clip at the position of the wide char. If we attempt to clip for the width of the wide
+            // char it introduces artifacts when a wide char is partially clipped.
+            if (CurrentClip.ContainsExclusive(position))
```

Comment:
If start position is clipped we still might want to render other parts which are not clipped. I guess this if should be moved to inside "else" block

Reply:
fixed